<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDialog</class>
 <widget class="QDialog" name="mainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapbender QGIS-Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="removeSectionComboBoxLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Please select the server configuration section to remove</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="removeSectionComboBox"/>
   </item>
   <item>
    <widget class="QPushButton" name="removeSelectedSectionButton">
     <property name="text">
      <string>Remove selected section</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="removeDialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
